src/hypothesis/internal/escalation.py

```diff
@@ -53,15 +54,26 @@ def accept(filepath):
 is_hypothesis_file = belongs_to(hypothesis)
 is_coverage_file = belongs_to(coverage)
 
+HYPOTHESIS_CONTROL_EXCEPTIONS = (
+    DeadlineExceeded, StopTest, UnsatisfiedAssumption
```

Comment:
I like this idea - the implementation will probably require some comments, but I think it's a good idea.  Note that `HypothesisControlException` will need to subclass only from `BaseException`, and thus child exception classes will need to inherit from `(HypothesisException, HypothesisControlException)`.

Reply:
I'm not sure having to use multiple inheritance is an improvement on using a tuple!

Reply:
More complex implementation, cleaner use... I'll leave it up to you, but my instinct is to encode this grouping in our exception hierarchy rather than an external collection.

Reply:
My inclination is that subclassing is bad, and that keeping it in the one place that actually uses this distinction *is* cleaner use, so I'd prefer to stick with this approach.